kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProtocolFilterStatusFactory.java

```diff
@@ -57,9 +58,16 @@ KafkaProtocolFilter newFalseConditionStatusPatch(KafkaProtocolFilter observedPro
 
     @Override
     KafkaProtocolFilter newTrueConditionStatusPatch(KafkaProtocolFilter observedProxy,
-                                                    Condition.Type type) {
+                                                    Condition.Type type, String checksum) {
```

Comment:
the `checksum` parameter is unused, did you want to include it in the condition?

Reply:
I see it's just due to the refactor to the superclass, I'm assuming it's to be implemented in another PR?

Reply:
Yeah thats the plan. One PR per reconciler.